src/runtime-rs/crates/hypervisor/Cargo.toml

```diff
@@ -43,6 +43,10 @@ safe-path = "0.1.0"
 crossbeam-channel = "0.5.6"
 tempdir = "0.3.7"
 
+qapi = { version = "0.14", features = [ "qmp", "qga", "async-tokio-all" ] }
+qapi-spec = "0.3.1"
+qapi-qmp = "0.14.0"
+
```

Comment:
Corresponding changes in `Cargo.lock` should be added to this commit as well.

Reply:
You're right, thanks, I'll fix that.

Reply:
... fixed.